src/plugins/presentation_panel/public/panel_component/presentation_panel_internal.tsx

```diff
@@ -106,6 +110,7 @@ export const PresentationPanelInternal = <
       aria-labelledby={headerId}
       data-test-embeddable-id={api?.uuid}
       data-test-subj="embeddablePanel"
+      {...contentAttrs}
```

Comment:
Move this to the `EuiPanel` because the props weren't getting passed through properly to the embeddable component - so, `data-render-complete` was never being set for the image embeddable (which caused test failures due to us `awaiting` these attributes when waiting for dashboards to load).